### ISSUE TITLE
Typedoc secondary entrypoints rendered on the docs hub

### ISSUE THREAD

**Issue by petertonysmith94** [FuelLabs/fuels-ts#3037]
### Summary

As part of the #2919 we added multiple entrypoints for typedocs. There are correctly _not_ rendered as a navigation item within Vitepress, however, on the docs hub, these are being displayed as [navigation items](https://docs.fuel.network/docs/fuels-ts/contract/src-index/).

<img width="317" alt="image" src="https://github.com/user-attachments/assets/e2c42db7-1e41-47ad-a9eb-aa95e0cee953">

We should remove these items from being displayed as navigation items on the docs hub.

**Comment by danielbate** [comment]
This should've been resolved by #2937. When was docs hub last updated?

**Comment by petertonysmith94** [comment]
I can see that it's not in the [`apps/docs/.typedoc/api-links.json`](https://github.com/FuelLabs/fuels-ts/blob/docs/apps/docs/.typedoc/api-links.json). Potentially the `docs-hub` uses the directory contents for constructing these navigation items (so it's a docs-hub issue).

I'd favour isolating the API docs and referencing it, like what was discussed in sync.

**Comment by maschad** [comment]
> I'd favour isolating the API docs and referencing it, like what was discussed in sync.

I agree as well, I think that would be the ideal solution rather than maintaining these API docs via a _hacky_ [postbuild script ](https://github.com/FuelLabs/fuels-ts/blob/master/apps/docs/scripts/typedoc-postbuild.ts) it seems like misappropriated engineering resources in my opinion.

**Comment by petertonysmith94** [comment]
Added the [following](https://github.com/FuelLabs/fuels-ts/issues/3058).

Would we want to close this or move the issue to the docs-hub?

**Comment by petertonysmith94** [comment]
Closed in favour of:
- https://github.com/FuelLabs/fuels-ts/issues/3058.
